plugins/drude/openmmapi/src/DrudeLangevinIntegrator.cpp

```diff
@@ -30,6 +30,7 @@
  * -------------------------------------------------------------------------- */
 
 #include "openmm/DrudeLangevinIntegrator.h"
+#include "SimTKOpenMMRealType.h"
```

Comment:
What is this for?  I don't think you added any references to things defined in this file.

Reply:
good catch, thanks